src/exceptions/u-boot-exception-2.0.xml

```diff
@@ -6,7 +6,7 @@
       </crossRefs>
       <notes>Typically used with GPL-2.0+</notes>
       <titleText>
-         <p>The U-Boot License Exception:</p>
+         <p>GPL License Exception:</p>
```

Comment:
I think we need to support the old form for compatibility with our 3.0 release, [2.6 release][1], and possibly earlier releases.  But I agree that our canonical text should match upstream.  That would mean markup like:

```xml
<p><alt name="title" match="GPL|The U-Boot">GPL</alt> License Exception:</p>
```

[1]: https://github.com/spdx/license-list-data/blob/v2.6/rdfa/u-boot-exception-2.0.html#L160